SecurityPkg/Library/DxeTcg2PhysicalPresenceLib/PhysicalPresenceStrings.uni

```diff
@@ -12,8 +12,18 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #string TPM_PPI_HEAD_STR              #language en-US    "A configuration change was requested to allow the Operating System to %s the computer's TPM (Trusted Platform Module) without asking for user confirmation in the future.\n\n"

```

Comment:
look at physical presence spec - this was potentially implemented for tablets. this may require spec change. look for a different commit / or make change that pushes this logic up to allow platforms to make changes. This may need to be a library allowing edk2 to use their own strings and a platform their own